### ISSUE TITLE
Docker no longer works

### ISSUE THREAD

**Issue by mmynsted** [docker/for-mac#2121]
### Expected behavior

Docker to start

### Actual behavior

Docker will not start.

### Information

`CD0342CE-C58B-44E8-A012-970412B8A427`

  - Full output of the diagnostics from "Diagnose & Feedback" in the menu
  - A reproducible case if this is a bug, Dockerfiles FTW
  - Page URL if this is a docs issue or the name of a man page

### Steps to reproduce the behavior

  1. upgrade or uninstall and re-install Docker
  2. Start docker
  3. Experience crash
  4. reset to factory default or do not.  (Either way)
  5. crash

**Comment by mmynsted** [comment]
Example of messages:

```
Docker for Mac: version: 17.09.0-ce-mac32 (69202b202f497d4b6e627c3370781b9e4b51ec78)
macOS: version 10.12.6 (build: 16G29)
logs: /tmp/CD0342CE-C58B-44E8-A012-970412B8A427/20171003-090207.tar.gz
failure: Docker.qcow2 missing: the VM has never been started
[OK]     db.git
[OK]     vmnetd
[OK]     dns
[ERROR]  driver.amd64-linux
         com.docker.driver.amd64-linux is not running
[OK]     virtualization VT-X
[OK]     app
[ERROR]  moby
         /Users/mmynsted/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/console-ring does not exist
[OK]     system
[OK]     moby-syslog
[ERROR]  db
         Connection refused (ECONNREFUSED) connecting to /Users/mmynsted/Library/Containers/com.docker.docker/Data/s40: check if service is running
         com.docker.db is not running
[OK]     env
[OK]     virtualization kern.hv_support
[ERROR]  slirp
         Unexpected error ((Failure
  "Error connecting socket to 9p endpoint unix:/Users/mmynsted/Library/Containers/com.docker.docker/Data/s51: Unix.Unix_error(Unix.ECONNREFUSED, \"connect\", \"\")")) connecting to /Users/mmynsted/Library/Containers/com.docker.docker/Data/s51
         com.docker.vpnkit is not running
         slirp check failed with: Unix.Unix_error(Unix.ECONNREFUSED, "connect", "")
[ERROR]  osxfs
         com.docker.osxfs is not running
[OK]     moby-console
[OK]     logs
[ERROR]  docker-cli
         Connection refused (ECONNREFUSED) connecting to /var/run/docker.sock: check if service is running
         Connection refused (ECONNREFUSED) connecting to /Users/mmynsted/Library/Containers/com.docker.docker/Data/s60: check if service is running
         docker ps failed
[OK]     menubar
[ERROR]  disk
         Docker.qcow2 missing: the VM has never been started
```

**Comment by gtardif** [comment]
This should be fixed with the latest version (17.09.0-ce-mac34 on edge channel, mac35 on stable channel)
